crates/biome_cli/src/changed.rs

```diff
@@ -7,14 +7,14 @@ use std::ffi::OsString;
 pub(crate) fn get_changed_files(
     fs: &DynRef<'_, dyn FileSystem>,
     configuration: &PartialConfiguration,
-    since: Option<String>,
+    since: Option<&String>,
```

Comment:
It seems more idiomatic to use `&str`. You can use `as_derefe()` at call site to turn `Option<String>` to `Option<&str>`.
```suggestion
    since: Option<&str>,
```

Reply:
It seems or it is? 😬

Reply:
&String is technically an extra indirection. Probably just a micro optimization.

Reply:
"It is" :)